packages/block-editor/src/components/block-mover/index.js

```diff
@@ -66,28 +70,54 @@ export class BlockMover extends Component {
 				} ) }
 			>
 				<ToolbarGroup>
-					<ToolbarItem
-						onFocus={ this.onFocus }
-						onBlur={ this.onBlur }
-					>
-						{ ( itemProps ) => (
-							<BlockMoverUpButton
-								clientIds={ clientIds }
-								{ ...itemProps }
-							/>
-						) }
-					</ToolbarItem>
-					<ToolbarItem
-						onFocus={ this.onFocus }
-						onBlur={ this.onBlur }
-					>
-						{ ( itemProps ) => (
-							<BlockMoverDownButton
-								clientIds={ clientIds }
-								{ ...itemProps }
-							/>
-						) }
-					</ToolbarItem>
+					<ToolbarGroup className="block-editor-block-mover__move-button-container">
```

Comment:
Is it semantically okay to nest a `ToolbarGroup` inside of another `ToolbarGroup`? One could argue the up/down buttons and the drag handle belong on the same level, though of course this complicates the styling somewhat (unless you're willing to remove _all_ special case styling that the mover buttons currently have, in which case the styling becomes really simple).

Reply:
This CSS suggests it was at least intended to work that way: https://github.com/WordPress/gutenberg/blob/master/packages/components/src/toolbar-group/style.scss#L9